pylint_odoo/test/main.py

```diff
@@ -247,7 +250,7 @@ def my_which(bin_name, *args, **kwargs):
         self.expected_errors.pop('javascript-lint')
         self.assertEqual(self.expected_errors, real_errors)
 
-    def test_90_valid_odoo_versions(self):
+    def test_90_valid_odoo_versions_8_0(self):
         """Test --valid_odoo_versions parameter when is '8.0'"""
         extra_params = ['--valid_odoo_versions=8.0',
                         '--disable=all',
```

Comment:
Shouldn't be enabled `useless-attribute-non-translatable` here to see that 8.0 don't emit this message?

Reply:
Good point, I enabled the check `useless-attribute-non-translatable` in the method `test_90_valid_odoo_versions_8_0`